src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -352,10 +353,20 @@ def _gather_requirements_files(
 
         # Generate requirements file for the local environment if configured
         if docker_settings.replicate_local_python_environment:
+            command: List[str]
```

Comment:
I just saw this, which can probably removed (or at least be a `str` which it gets set too later on)